app/views/bulk_updates/show.html.erb

```diff
@@ -0,0 +1,41 @@
+<div class="row row-content">
+  <section>
+    <h2>
+      Bulk Update <%= @task_id %>
+      <span class="bulk-update-status eui-badge--sm"><%= @task.fetch('task-status') %></span>
+    </h2>
+    <p class="bulk-update-status-message"><%= @task.fetch('status-message', 'Status not retrieved') %></p>
+
+  </section>
+
+  <section>
+    <% unless @task.fetch('collection-statuses', nil).blank? %>
+      <h4>Bulk Update Errors</h4>
+      <table class="space-top space-bot" id="bulk-update-errors-table">
+        <thead>
+          <tr>
+            <th>Entry Title</th>
+            <th>Short Name</th>
+            <th>Version</th>
+            <th>Status Message</th>
+          </tr>
+        </thead>
+
+        <tbody>
+          <% @task['collection-statuses'].each do |task_error| %>
```

Comment:
haha :). I think it seemed ok because of the check above to get into the table, but yes there could still be some issue with trying to iterate over the response.